src/tests/system/tests/test_ldap.py

```diff
@@ -144,10 +144,10 @@ def test_ldap__password_change_new_password_does_not_meet_complexity_requirement
         "user1", "Secret123", "red_32"
     ), "Password should not have been able to be changed!"
 
-    assert (
-        "pam_sss(passwd:chauthtok): User info message: Password change failed."
-        in client.host.conn.run("journalctl").stdout
+    log = client.journald.journalctl(
+        grep="pam_sss(passwd:chauthtok): User info message: Password change failed.", unit="sssd"
     )
+    assert log.rc == 0, "'Password change failed.' message is not in log!"
```

Comment:
Patrik and Dan added `is_match` that returns bool for this case:

```
match = client.journald.is_match("pam_sss(passwd:chauthtok): User info message: Password change failed.", unit="sssd")
assert match, msg
```

Reply:
done